README.md

```diff
@@ -2,6 +2,8 @@
 
 [![Language](https://img.shields.io/badge/python-3.5%2B-blue.svg)](https://www.python.org/)
 [![License](https://img.shields.io/badge/license-GPL-yellow.svg)](https://github.com/aria-tools/ARIA-tools/blob/master/LICENSE)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/b69726c41ff9498882088afe57fe8391)](https://app.codacy.com/app/ehavazli/ARIA-tools?utm_source=github.com&utm_medium=referral&utm_content=ehavazli/ARIA-tools&utm_campaign=Badge_Grade_Settings)
```

Comment:
This is pointing to your own app for codacy and the circleci

Reply:
I made the changes